src/modules/airdrop/services/airdrop-service.ts

```diff
@@ -268,10 +279,17 @@ export class AirdropService {
           }
         : null,
     };
+
+    await this.cacheManager.set(cacheKey, data, 300);
```

Comment:
I'd recommend putting the two 300 hardcoded values into an easier to change variable

Reply:
yeah, you're right